src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -616,6 +616,34 @@ namespace cryptonote
     return true;
   }
   //-----------------------------------------------------------------------------------------------
+  std::pair<uint64_t, uint64_t> core::get_coinbase_tx_sum(const uint64_t start_offset, const uint64_t count)
+  {
+    std::list<block> blocks;
+    std::list<transaction> txs;
+    std::list<crypto::hash> missed_txs;
+    uint64_t coinbase_amount = 0;
+    uint64_t emission_amount = 0;
+    uint64_t total_fee_amount = 0;
+    uint64_t tx_fee_amount = 0;
+    this->get_blocks(start_offset, count, blocks);
```

Comment:
This is also likely to take a LOT of memory. The database keeps track of the coins emitted up to a given block. Subtracting those on the end/start blocks will be a lot faster and less memory hungry.